lib/osf-components/addon/components/institutions-widget/template.hbs

```diff
@@ -18,18 +18,15 @@
         <modal.body>
             <InstitutionSelectList
                 @model={{this.currentUser.user}}
```

Comment:
Could you rename `@model` to `@user`? `model` isn't very clear outside of a route template.